README.md

```diff
@@ -51,24 +51,34 @@ Things you may want to cover:
 
     a. [PostgreSQL](https://www.postgresql.org/download/)
 
+5. Make sure you the yarn package manager installed by running: `npm i -g yarn`
+
+   a. `yarn` is a great alternative to `npm` as a dependency manager for Node.js. Read more [here](https://yarnpkg.com/en/).
+
 ## Setup
 
 __Disclaimer:__ Windows users, it is highly recommended that the entire setup process should be done through the Windows Subsystem for Linux (WSL). Setting up WSL can be found [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
 
-1. Make sure you the yarn package manager installed by running: `npm i -g yarn`
+__Pre-req:__ As mentioned in [configuration](#Configuration), you must recieve the `.env` file from GTHC project leaders before beginning setup process.
 
-   a. `yarn` is a great alternative to `npm` as a dependency manager for Node.js. Read more [here](https://yarnpkg.com/en/).
+1. Install all packages/dependancies.
+    
+    a. Run `bundle install`
+    
+    b. Run `yarn install`
 
 2. `yarn run setup`
 
    a. This yarn script should run all of the necessary shell scripts to setup your Docker containers, bundle (ruby gems) dependencies, and your node dependencies.
 
 3. `yarn dev:start`
 
-    a. This will start a local developer server, by default that will be on port 5000. (`https://localhost:5000`)
+    a. This will start a local developer server, by default that will be on port 5000. (`http://localhost:5000`)
+    
+    b. Always run this command from here on out to start the local server.
 
 ### Local Software Versioning
 It is required that your system is running `Rails 5.0+` for the local development to function properly. It is recommended to use the most recent version of Node, but anything above `Node 8.x` shall work. 
 
 ## Configuration
-__Important:__ There are private environment variables needed for certain aspects of the application. Therefore, it will be required that you have a `.env` file in the root of your project __before__ running anything. Please contact the project leaders (Aman or Anesu) for this information. __Finally, the .env file should not be pushed to github under no circumstances!__
+__Important:__ There are private environment variables needed for certain aspects of the application. Therefore, it will be required that you have a `.env` file in the root of your project __before__ running anything. Please contact the project leaders (Aman or Anesu) for this information. __Finally, the .env file should not be pushed to github under no circumstances!__
```

Comment:
I know you provided the updated .env file in the dev chat, but maybe mention that an updated .env file is required in the readme?